main.py

```diff
@@ -2,40 +2,36 @@
 import glob
 import re
 import os
-# Ścieżka do folderu, w którym znajdują się pliki
-folder_path = os.getcwd() + '\data'
-
+# Path to the folder where the files are located
+folder_path = os.path.join(os.getcwd(), 'data')
+file_extension = ['xlsx', 'csv', 'txt']
+files = []
+Columns = ["Questioned_Date", "Model_No", "OS", "SW_Version", "CSC", "Category", "Application_Name", "content"]

```

Comment:
First capital letter in a variable dentoes it's a class object, for global variables it's good practice to use all capital letters